src/defs.ts

```diff
@@ -139,3 +139,16 @@ export interface DomainsPage {
 export interface Permissions {
   permissions: string[]
 }
+
+export interface Invitation {
```

Comment:
An invitation struct looks like this:
```
type Invitation struct {
	InvitedBy   string    `json:"invited_by"`
	UserID      string    `json:"user_id"`
	DomainID    string    `json:"domain_id"`
	Token       string    `json:"token,omitempty"`
	Relation    string    `json:"relation,omitempty"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at,omitempty"`
	ConfirmedAt time.Time `json:"confirmed_at,omitempty"`
	Resend      bool      `json:"resend,omitempty"`
}

```
Add missing fields and remove `metadata` field.

Also update the `User` interface to include `created_at` field

Reply:
also use snake case

Reply:
For relation use type `Relation`